### PR TITLE
Add questions results page

### PR DESCRIPTION
**Pull request by sah-shah7**
-Added Questions Page
image

-Added Results Page
image

### CODE REVIEW

File: stylesheets/categories-css/category-1/questions.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.header-question{
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 1rem;
+}
+
+.body-question{
+    margin: 1rem 0 1rem 0;
```
Comment (NeerajVodala) [inline stylesheets/categories-css/category-1/questions.css:16]: can make it shorthand (margin: 1rem 0;)

File: stylesheets/categories-css/category-1/questions.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+.ans{
+    outline: none;
+    border:none;
+    padding: 1rem 1.5rem 1rem 1.5rem;
```
Comment (NeerajVodala) [inline stylesheets/categories-css/category-1/questions.css:21]: shorthand here too (padding: 1rem 1.5rem)

**Review by NeerajVodala — APPROVED** [review]
Looks good(added few suggestions in comments).
All the Best.

**Comment by sah-shah7** [comment]
PR Approved. Merging

**Pull request merged**